src/views/Simulation/components/TopBar/SettingsButton.js

```diff
@@ -191,6 +191,16 @@ export const SettingsButton = () => {
         {outputLevelsSelect}
       </Stack>
     );
+    const filterEffectsSettings = (
```

Comment:
The "Settings" suffix is used above for a `Stack` of several components, but this one only has one `Switch` toggle. 
Can we rename it to `glowEffectSwitch`?